source/Cosmos.System2/Graphics/FullScreenCanvas.cs

```diff
@@ -66,7 +66,7 @@ private static Canvas GetVideoDriver()
             {
                 return new SVGAIICanvas();
             }
-            else if (BGAExists() || PCI.GetDevice((VendorID)0x80EE, (DeviceID)0xBEEF) != null || Core.VBE.IsAvailable())
+            else if (BGAExists() || PCI.Exists(VendorID.VirtualBox, DeviceID.VBVGA) || PCI.Exists(VendorID.Bochs, DeviceID.BGA) || Core.VBE.IsAvailable())
```

Comment:
Can we combine all of these into a single VBECanvasAvailable?

Reply:
Yeah, I thought I could do that